torch/_dynamo/compiled_autograd.py

```diff
@@ -207,6 +209,24 @@ def end_capture(self, outputs):
         )
         return self.compiler_fn(graph)
 
+    def reorder_accumulate_grad_nodes(self):
+        """
+        Usage of AOTAutograd causes all the accumulate_grad_ nodes to get pushed to the end of
```

Comment:
Running AOTAutograd again on the compiled autograd graph won't cause this?

Reply:
The issue I am trying to fix here is ordering with respect to hooks.